tests/api2/test_005_interface.py

```diff
@@ -60,8 +61,16 @@ def get_payload(ws_client):
 
     return payload, to_validate
 
+# Make sure that our initial conditions are met
+def test_001_check_ipvx(request):
+    # Verify that dhclient is running
+    running, _ = call('interface.dhclient_status', interface)
+    assert running
```

Comment:
Also better to be explicit about the assert i.e `assert running is True` (assuming `running` should be a boolean)